sdk/bpf/c/bpf.mk

```diff
@@ -222,6 +222,7 @@ endef
 
 define TEST_EXEC_RULE
 $1: $2
+	LD_LIBRARY_PATH=$(TESTFRAMEWORK_RPATH) \
```

Comment:
@dmakarov I didn't find where the criterion rpath is setup for normal CI runs.  This seems like a safe, non-intrusive alternative I think?

Reply:
yes, looks good to me.

Reply:
Thanks for taking a look!